lib/_footer.erb

```diff
@@ -104,6 +104,9 @@
               });
             </script>
           <%- end -%>
+        <%- elsif categories.key?("HubSpot") && !categories["HubSpot"].nil? -%>
```

Comment:
I thought as this is the same change done on the other branch, it wouldn't affect anything when we cherry pick and merge. If otherwise, I can just edit the commit to just keep the change on the yml file.